content/en/real_user_monitoring/browser/setup/server.md

```diff
@@ -0,0 +1,135 @@
+---
+title: RUM Browser Monitoring Server Side Instrumentation
+beta: true
+code_lang: server
+type: multi-code-lang
+code_lang_weight: 2
+further_reading:
+- link: '/real_user_monitoring/explorer/'
+  tag: 'Documentation'
+  text: 'Learn about the RUM Explorer'
+- link: '/logs/log_collection/javascript/'
+  tag: 'Documentation'
+  text: 'Learn about the Datadog Browser SDK for Logs'
+---
+
+{{< callout url="https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/" btn_hidden="false" header="Access the Preview!">}}
+RUM SDK Injection in Preview. Sign up for access! You can also request early access if you are using Apache HTTPD, Tomcat, or another web server technology.
+{{< /callout >}}
+
+## Overview
+
+Datadog RUM Server Side Instrumentation (SDK Injection) lets you opt into Real User Monitoring (RUM) automatically by instrumenting web applications served through a web server or proxy.
+
+RUM Server Side Instrumentation works by injecting a RUM SDK JavaScript scriptlet into the HTML responses being served through a web server or proxy.
+
+After your applications have been instrumented, you can begin configuring your RUM application in Datadog.
+
+## Prerequisites
+
+The automatic installation method requires that you have the [Datadog Agent][2] installed.
+
+## Setup your RUM application
+
+<div class="alert alert-warning">To request support for a web server that is not listed here, <a href="https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/">fill out this form.</a></div>
+
+{{< tabs >}}
+{{% tab "Nginx" %}}
+
+The Datadog RUM Server Side Instrumentation method leverages the [Nginx Dynamic Modules capability][1] to implement a response body filter. The filter injects the RUM SDK into the response body for responses identified as HTML.
+
+To automatically instrument your RUM application:
+
+1. In Datadog, navigate to the [**Digital Experience > Add an Application Page**][2] and select the JavaScript (JS) application type.
+2. Select **Auto-Instrumentation** and **NGINX**.
+3. Set your Session and Session Replay sample rates. See [guidance on configuring sampling][3].
+4. Copy and run the installer command to load the Datadog RUM SDK Injector onto your Nginx module.
+5. After the installer successfully installs the SDK Injector, restart your Nginx to begin collecting RUM sessions.
```

Comment:
```suggestion
5. After the installer successfully installs the SDK Injector, restart Nginx to begin collecting RUM sessions.
```
"your" is not grammatically wrong, just sounded a bit odd to me in this context.